packages/react/src/components/Rating/Rating.base.tsx

```diff
@@ -109,7 +110,8 @@ export const RatingBase: React.FunctionComponent<IRatingProps> = React.forwardRe
       theme: theme!,
     });
 
-    const ariaLabel = getAriaLabel?.(displayRating, max);
+    const normalModeAriaLabel = ariaLabel ? ariaLabel : getAriaLabel?.(displayRating, max);
+    const readOnlyAriaLabel = getAriaLabel?.(displayRating, max);
```

Comment:
If you switch the order of these, you only need to call `getAriaLabel` once:

```js
const readOnlyAriaLabel = getAriaLabel?.(displayRating, max);
const normalModeAriaLabel = ariaLabel ? ariaLabel : readOnlyAriaLabel;
```

Reply:
Yup! Total oversight on my part